tests/Unit/CompanyTest.php

```diff
@@ -0,0 +1,375 @@
+     */
+    public function testItHasInformation()
+    {
+        $company = $this->company(self::TEST_COMPANY);
+
+        $this->assertIsString($company->getInformation());
+    }
+
+    /**
+     * @throws \Http\Client\Exception
+     * @throws \Phpfastcache\Exceptions\PhpfastcacheInvalidArgumentException
+     * @throws \TruckersMP\Exceptions\PageNotFoundException
+     * @throws \TruckersMP\Exceptions\RequestException
+     */
+    public function testItHasRules()
+    {
+        $company = $this->company(self::TEST_COMPANY);
+
+        $this->assertIsString($company->getRules());
+    }
+
+    /**
+     * @throws \Http\Client\Exception
+     * @throws \Phpfastcache\Exceptions\PhpfastcacheInvalidArgumentException
+     * @throws \TruckersMP\Exceptions\PageNotFoundException
+     * @throws \TruckersMP\Exceptions\RequestException
+     */
+    public function testItHasRequirements()
+    {
+        $company = $this->company(self::TEST_COMPANY);
+
+        $this->assertIsString($company->getRequirements());
+    }
+
+    /**
+     * @throws \Http\Client\Exception
+     * @throws \Phpfastcache\Exceptions\PhpfastcacheInvalidArgumentException
+     * @throws \TruckersMP\Exceptions\PageNotFoundException
+     * @throws \TruckersMP\Exceptions\RequestException
+     */
+    public function testItHasAWebsite()
+    {
+        $company = $this->company(self::TEST_COMPANY);
+
+        $this->assertIsString($company->getWebsite());
+    }
+
+    /**
+     * @throws \Http\Client\Exception
+     * @throws \Phpfastcache\Exceptions\PhpfastcacheInvalidArgumentException
+     * @throws \TruckersMP\Exceptions\PageNotFoundException
+     * @throws \TruckersMP\Exceptions\RequestException
+     */
+    public function testItHasSocialInformation()
+    {
+        $company = $this->company(self::TEST_COMPANY);
+
+        $this->assertInstanceOf(Social::class, $company->getSocial());
+
+        // Twitter
+        if ($company->getSocial()->getTwitter() !== null) {
+            $this->assertIsString($company->getSocial()->getTwitter());
+        }
+
+        // Facebook
+        if ($company->getSocial()->getFacebook() !== null) {
+            $this->assertIsString($company->getSocial()->getFacebook());
+        }
+
+        // Twitch
+        if ($company->getSocial()->getTwitch() !== null) {
+            $this->assertIsString($company->getSocial()->getTwitch());
+        }
+
+        // PlaysTV
+        if ($company->getSocial()->getPlays() !== null) {
+            $this->assertIsString($company->getSocial()->getPlays());
+        }
+
+        // Discord
+        if ($company->getSocial()->getDiscord() !== null) {
+            $this->assertIsString($company->getSocial()->getDiscord());
+        }
+
+        // YouTube
+        if ($company->getSocial()->getYouTube() !== null) {
+            $this->assertIsString($company->getSocial()->getYouTube());
+        }
+
+        $this->assertInstanceOf(Social::class, $company->getSocial());
```

Comment:
This is also checked at the beginning of this method.

Reply:
Oops, will go ahead and remove